l10n_th_gov_work_acceptance/models/work_acceptance_committee.py

```diff
@@ -91,6 +91,7 @@ def create(self, vals):
                 and not work_acceptance.validated
                 and not work_acceptance.rejected
                 and not self._check_allow_write_under_validation(vals)
+                and not self._context.get("skip_validation_check")
```

Comment:
```suggestion
                and not self.env.context.get("skip_validation_check")
```

In version19+. We will use `self.env.context` instead of `self._context`
Should we change it to long support?

ref: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-19.0#tasks-to-do-in-the-migration